tests/integration/test_events.py

```diff
@@ -27,6 +27,8 @@
     'Detail': [EVENT_DETAIL]
 }
 
+TEST_EVENT_PATTERN_1 = {'detail': {'EventType': ['0', '1']}}
```

Comment:
nitpick: I'd rather move this into the test method directly (we should avoid variables at the root scope if they are not reused in multiple places). (not critical, though - shouldn't hold back the merge..)